io/js/src/main/scala/scalaz/effect/SafeApp.scala

```diff
@@ -6,5 +6,5 @@ trait SafeApp {
    * The main function of the application, which will be passed the command-line
    * arguments to the program.
    */
-  def run[E](args: List[String]): IO[E, Unit]
+  def run[E](args: List[String]): IO[Nothing, Unit]
```

Comment:
Extraneous type parameter.

Reply:
It is the same change we did for the jvm app, it should be `Void` but we ain't have it no more :smile:

Reply:
Sure but you can delete the `E` type parameter because it's not used on the right hand side.

Reply:
Oh you right, totally overlooked that